dashboards/nvidia-gpu/nvidia-triton-prometheus.json

```diff
@@ -616,81 +600,43 @@
         "width": 24,
         "height": 16,
         "widget": {
-          "title": "Inference latency by percentile",
           "xyChart": {
             "dataSets": [
```

Comment:
consider keeping the legend templates here?

Reply:
Good call, done (and updated the screenshot too). I don't remember why legend templates don't show up in the UI for PromQL queries but you can set them in the JSON.